### PR TITLE
Add social handles

### PR DESCRIPTION
**Pull request by sjanssen2**
Addressing #3404, I have extended the user profile edit page with three additional input fields to store social network handles for google scholar, research gate and orcid.
image
becomes:
image

Note that I have added WTForm validators to check user input and provide hints if he/she provides wrong inputs. That required a bit of changes in the HTML template as well as in the UserProfileHandler.post method. I've also added small icons for the social networks, which made be use CSS flex box mechanisms to properly align the icon with the label (are we legally allowed to display those?).

Since qiita already uses WTForm mechanism, I opted to simple extend the qiita.qiita_user table by three additional columns, namely those defined by adding additional `StringField` elements. Everything else (storing, retrieving, updating) came out of the box with existing code.

### CODE REVIEW

**Comment by sjanssen2** [comment]
@antgonza do you have an idea why my PR errors with `ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1131)`?

File: qiita_db/support_files/qiita-db-unpatched.sql [hunk 1]
```diff
@@ -1888,7 +1888,10 @@ CREATE TABLE qiita.qiita_user (
     user_verify_code character varying,
     pass_reset_code character varying,
     pass_reset_timestamp timestamp without time zone,
-    receive_processing_job_emails boolean DEFAULT false
+    receive_processing_job_emails boolean DEFAULT false,
+    social_orcid character varying,
+    social_researchgate character varying,
+    social_googlescholar character varying
```
Comment (antgonza) [inline qiita_db/support_files/qiita-db-unpatched.sql:1894]: Can you add DEFAULT NULL?

File: qiita_db/support_files/populate_test_db.sql [hunk 2]
```diff
@@ -50,10 +50,10 @@ INSERT INTO qiita.user_level VALUES (7, 'wet-lab admin', 'Can access the private
...
-INSERT INTO qiita.qiita_user VALUES ('shared@foo.bar', 4, '$2a$12$gnUi8Qg.0tvW243v889BhOBhWLIHyIJjjgaG6dxuRJkUM8nXG9Efe', 'Shared', 'Nowhere University', '123 fake st, Apt 0, Faketown, CO 80302', '111-222-3344', NULL, NULL, NULL, false);
-INSERT INTO qiita.qiita_user VALUES ('admin@foo.bar', 1, '$2a$12$gnUi8Qg.0tvW243v889BhOBhWLIHyIJjjgaG6dxuRJkUM8nXG9Efe', 'Admin', 'Owner University', '312 noname st, Apt K, Nonexistantown, CO 80302', '222-444-6789', NULL, NULL, NULL, false);
-INSERT INTO qiita.qiita_user VALUES ('demo@microbio.me', 4, '$2a$12$gnUi8Qg.0tvW243v889BhOBhWLIHyIJjjgaG6dxuRJkUM8nXG9Efe', 'Demo', 'Qiita Dev', '1345 Colorado Avenue', '303-492-1984', NULL, NULL, NULL, false);
+INSERT INTO qiita.qiita_user VALUES ('test@foo.bar', 4, '$2a$12$gnUi8Qg.0tvW243v889BhOBhWLIHyIJjjgaG6dxuRJkUM8nXG9Efe', 'Dude', 'Nowhere University', '123 fake st, Apt 0, Faketown, CO 80302', '111-222-3344', NULL, NULL, NULL, false, '0000-0002-0975-9019', 'Rob-Knight', '_e3QL94AAAAJ');
```
Comment (antgonza) [inline qiita_db/support_files/populate_test_db.sql:53]: If you add "DEFAULT NULL" to the table these changes are not needed.

**Review by antgonza — CHANGES_REQUESTED** [review]
Thank you, this looks good. Some minor comments.

**Comment by antgonza** [comment]
Yup, we just merged the fix for that and these tests should run fine.

**Comment by antgonza** [comment]
Looks like you have to pull from dev and then push; could you do that?

File: qiita_db/support_files/qiita-db-unpatched.sql (hunk 1 above)
Comment (sjanssen2) [reply qiita_db/support_files/qiita-db-unpatched.sql]: done

File: qiita_db/support_files/populate_test_db.sql (hunk 2 above)
Comment (sjanssen2) [reply qiita_db/support_files/populate_test_db.sql:53]: I removed the NULLs

**Comment by sjanssen2** [comment]
Hi @antgonza I made the requested changes and added some tests. Can you give this PR another review?

File: qiita_pet/static/img/logo_social_googlescholar.png
Comment (charles-cowart) [inline qiita_pet/static/img/logo_social_googlescholar.png:1]: legally these are all right to use and store in the repo?

**Review by charles-cowart — CHANGES_REQUESTED** [review]
looks good! just need confirmation on one item.

File: qiita_pet/static/img/logo_social_googlescholar.png
Comment (sjanssen2) [reply qiita_pet/static/img/logo_social_googlescholar.png:1]: Here are the sources for the three icons:

- google: https://iconduck.com/icons/14098/google-scholar "is licensed under the open source [MIT](https://iconduck.com/licenses/mit) license"
- research gate: https://commons.wikimedia.org/wiki/File:ResearchGate_icon_SVG.svg "The person who associated a work with this deed has dedicated the work to the [public domain](https://en.wikipedia.org/wiki/en:public_domain) by waiving all of their rights to the work worldwide under copyright law, including all related and neighboring rights, to the extent allowed by law. You can copy, modify, distribute and perform the work, even for commercial purposes, all without asking permission."
- https://iconduck.com/icons/14166/orcid "is licensed under the open source [MIT](https://iconduck.com/licenses/mit) license"

I read this as we are allowed to store in our repo

**Review by charles-cowart — APPROVED** [review]

**Pull request merged**